app/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1,15 @@
+//
+// Variables
+// --------------------------------------------------
+
+
+//== Colors
+//
+//## Gray colors for use across Coursemology.
+
+$gray-base:              #000 !default;
```

Comment:
![lol](http://www.quickmeme.com/img/2c/2ca7e6a17875890e4c6f4924fc20df7ae8919482f870f8edc03005674896394e.jpg)

Let's stick to American.